port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -273,15 +273,16 @@ async def _parse_items(
             if len(result.misconfigurations) > 0:
                 entity_misconfigurations |= result.misconfigurations
 
+            if (
+                len(examples_to_send) < send_raw_data_examples_amount
+                and result.raw_data is not None
+            ):
+                examples_to_send.append(result.raw_data)
+
```

Comment:
I think we do would like to send the passed entities first if they are passing, rather than those filtered out. 
e.g. 

if the first 10 are being filtered out, i wouldn't want to send them, but rather send the passed ones first. 
let me know if you find a creative and a clean way to do so 💭  as if it implies not very clean solution than we might pass on my suggestion